tests/data/extract_coding/SRR306838_GSM752691_hsa_br_F_1_trimmed_subsampled_n22__alphabet-dayhoff_ksize-12.csv

```diff
@@ -1,17 +1,17 @@
 read_id,jaccard_in_peptide_db,n_kmers,classification,filename
 SRR306838.10559374 Ibis_Run100924_C3PO:6:51:17601:17119/1,1.0,11.0,Coding,SRR306838_GSM752691_hsa_br_F_1_trimmed_subsampled_n22.fq
 SRR306838.6196593 Ibis_Run100924_C3PO:6:29:16733:12435/1,0.08333333333333333,12.0,Non-coding,SRR306838_GSM752691_hsa_br_F_1_trimmed_subsampled_n22.fq
-SRR306838.20767303 Ibis_Run100924_C3PO:6:104:6864:5062/1,0.09090909090909093,11.0,Non-coding,SRR306838_GSM752691_hsa_br_F_1_trimmed_subsampled_n22.fq
+SRR306838.20767303 Ibis_Run100924_C3PO:6:104:6864:5062/1,0.09090909090909091,11.0,Non-coding,SRR306838_GSM752691_hsa_br_F_1_trimmed_subsampled_n22.fq
 SRR306838.12582274 Ibis_Run100924_C3PO:6:62:11779:17975/1,0.0,11.0,Non-coding,SRR306838_GSM752691_hsa_br_F_1_trimmed_subsampled_n22.fq
 SRR306838.13334230 Ibis_Run100924_C3PO:6:66:16579:20350/1,0.0,7.0,Non-coding,SRR306838_GSM752691_hsa_br_F_1_trimmed_subsampled_n22.fq
 SRR306838.2740879 Ibis_Run100924_C3PO:6:13:11155:5248/1,1.0,13.0,Coding,SRR306838_GSM752691_hsa_br_F_1_trimmed_subsampled_n22.fq
 SRR306838.6813354 Ibis_Run100924_C3PO:6:32:10591:13073/1,0.0,12.0,Non-coding,SRR306838_GSM752691_hsa_br_F_1_trimmed_subsampled_n22.fq
-SRR306838.23113368 Ibis_Run100924_C3PO:6:114:13840:18459/1,0.0,9.0,Non-coding,SRR306838_GSM752691_hsa_br_F_1_trimmed_subsampled_n22.fq
-SRR306838.10872941 Ibis_Run100924_C3PO:6:53:6164:10522/1,0.09090909090909093,11.0,Non-coding,SRR306838_GSM752691_hsa_br_F_1_trimmed_subsampled_n22.fq
+SRR306838.23113368 Ibis_Run100924_C3PO:6:114:13840:18459/1,0.0,10.0,Non-coding,SRR306838_GSM752691_hsa_br_F_1_trimmed_subsampled_n22.fq
```

Comment:
the csv hasn't changed much except for these values after changing the function to yield every time. does this csv look right to you? i think I am still missing what the ground truth should be

Reply:
I haven't changed any input files intentionally